### ISSUE TITLE
Failing to load MDN specs

### ISSUE THREAD

**Issue by npm1** [tabatkins/bikeshed#2045]
Looks like Bikeshed is failing to load MDN changes, and fails with: `Couldn't load MDN Spec Links data for this spec.`

Example failures:
https://github.com/WICG/layout-instability/runs/2493650275?check_suite_focus=true (GH Actions)
https://github.com/WICG/layout-instability/pull/103#issue-628321015 (PR preview)

**Comment by birtles** [comment]
I'm seeing the same thing on web-animations-1 after updating bikeshed recently.

Adding `Include Mdn Panels: no` to the metadata seems to work around it for now.

**Comment by tabatkins** [comment]
Sigh, sorry about this, last time this happened it was a problem with the upstream data. I'll figure out what's causing problems now.

**Comment by tabatkins** [comment]
FYI, it turns out the data source I was using for these panels hasn't been maintained in years, which is part of the problem. (The other problem is that I wasn't testing that the file I fetched was valid JSON, so I was caching the HTML 404 response as the Nesting data file.)

**Comment by romainmenke** [comment]
Maybe https://github.com/web-platform-dx/web-features can be the new data source?

**Comment by tabatkins** [comment]
No, I just want to consume <https://github.com/mdn/browser-compat-data> directly; it's what the bot was doing previously (just organizing the firehose into individual files for me)
